reporting/tests/seed.py

```diff
@@ -87,21 +84,20 @@ def add_code(scenario, project_id, file_path_override=None):
                 "repo_url": "",
             }
 
-            code_response = requests.post(
-                TDS_URL + "/code",
+            code_response = auth_session().post(
+                TDS_URL + "/code-asset",
                 json=payload,
-                headers={"Content-Type": "application/json"},
             )
 
             if code_response.status_code >= 300:
                 raise Exception(
                     f"Failed to POST code ({code_response.status_code}): {scenario}"
                 )
             else:
-                add_asset(_id, "code", project_id)
+                add_asset(code_response.json()['id'], "code", project_id)
```

Comment:
Unfortunately I think that if we are getting a dynamic uuid back from the HMI server here the rest of the tests will break. I believe they are still dependent on the ids being specified by the directory structure in the tests. So when we generate the id above in the old code with `_id = f"{project_id}-{scenario.replace(' ','-')}"` this is also the expected id structure further down the line for the rest of the tests.  Any id changes that don't follow this expectation may break the tests, but I would have to run this new code to be sure.

Reply:
I think that because you mock TDS calls, tests will still pass